examples/demo/job.properties

```diff
@@ -38,3 +38,5 @@ oozie.use.system.libpath=true
 # TODO: replace it with wf:user once we implement it
 userName=${user.name}
 oozie.wf.application.path=/user/${user.name}/${examplesRoot}/apps/demo
+oozie.wf.workflow.notification.url=http://35.234.110.167:8080/workflow?job-id=$jobId&status=$status
+oozie.wf.action.notification.url=http://35.234.110.167:8080/action?job-id=$jobId&node-name=$nodeName&status=$status
```

Comment:
```suggestion
oozie.wf.action.notification.url=http://www.example.com:8080/action?job-id=$jobId&node-name=$nodeName&status=$status
```

Reply:
Fixed in 57fd9a185f161ca802aa7aa6e158f4132b911c70